<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Algorithm WIndow</string>
    </property>
    <widget class="QTreeWidget" name="algorithmWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>256</width>
       <height>621</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Algorithms</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Smoothing</string>
      </property>
      <item>
       <property name="text">
        <string>Normal Bluring</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaussian Bluring</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Median Bluring</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>HoughCircle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>findCircle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Threshold</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sharpen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Histogram</string>
      </property>
      <item>
       <property name="text">
        <string>Histogram Calculation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Histogram Equalization</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Edge Filter</string>
      </property>
      <item>
       <property name="text">
        <string>CannyEdge</string>
       </property>
      </item>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>300</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Project Queue</string>
    </property>
    <widget class="QListWidget" name="queueListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>300</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameter</string>
    </property>
    <widget class="QWidget" name="paramWidget" native="true">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>20</y>
       <width>291</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>341</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Input WIndow</string>
    </property>
    <widget class="QPushButton" name="roiButton">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>20</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>340</width>
       <height>260</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>509</y>
       <width>321</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>input method</string>
     </property>
     <widget class="QPushButton" name="openButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="fileRadio">
         <property name="text">
          <string>File open</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="camRadio">
         <property name="text">
          <string>Camera open</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>341</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>605</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>RUN</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="text">
         <string>STEP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
